python-package/setup.py

```diff
@@ -16,7 +16,14 @@
 libpath = {'__file__': libpath_py}
 exec(compile(open(libpath_py, "rb").read(), libpath_py, 'exec'), libpath, libpath)
 
-LIB_PATH = [os.path.relpath(libfile, CURRENT_DIR) for libfile in libpath['find_lib_path']()]
+LIB_PATH = []
+for libfile in libpath['find_lib_path']():
+  try:
+    relpath = os.path.relpath(libfile, CURRENT_DIR)
+    LIB_PATH.append(relpath)
```

Comment:
```python
break
```
?

Reply:
Why?

Reply:
Because [the first path seem to be the only path we need](https://github.com/dmlc/xgboost/blob/master/python-package/xgboost/libpath.py#L24 ).

Reply:
I think `find_lib_path()` is guaranteed to return only existing library files. Okay then, I'll add `break` to prevent duplicates.